sdk/data/aztables/go.mod

```diff
@@ -4,8 +4,6 @@ go 1.16
 
 replace github.com/Azure/azure-sdk-for-go/sdk/azidentity => ../../azidentity
```

Comment:
This will fail your new check.

Reply:
Yes, but it will be updated after the azidentity release and should pass